app/k9mail/src/main/java/com/fsck/k9/account/AccountSetupFinishedLauncher.kt

```diff
@@ -0,0 +1,20 @@
+package com.fsck.k9.account
+
+import android.content.Context
+import app.k9mail.feature.account.setup.domain.ExternalContract
+import com.fsck.k9.Preferences
+import com.fsck.k9.activity.MessageList
+
+class AccountSetupFinishedLauncher(
+    private val context: Context,
+    private val preferences: Preferences,
+) : ExternalContract.AccountSetupFinishedLauncher {
+    override fun launch(accountUuid: String) {
+        val account = preferences.getAccount(accountUuid)
```

Comment:
This call has the potential to perform disk I/O. So we should only call it from a background thread.